.changeset/tidy-tigers-bow.md

```diff
@@ -0,0 +1,12 @@
+---
+'@sl-design-system/grid': minor
+---
+
+Grid improvements:
+- Add "Cancel selection" button to the bulk action toolbar
+- Add `column` argument to `GridColumnHeaderRenderer` type
+- Fix missing aria-label on a selection column checkbox
+- Fix styling so a small button does not expand the header row above 48px
+- Fix styling so a row has a height of 40px
```

Comment:
It looks like there is sth wrong, in this example a row has 37px height

<img width="1987" alt="image" src="https://github.com/user-attachments/assets/7aa518c9-7e30-46fa-9d0b-91c4e40e2f7d" />

Reply:
Fixed